jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-maintenance-pull.yaml

```diff
@@ -44,7 +44,11 @@
         repo-name: 'k8s.io/test-infra'
     - janitor:
         branch: master
-        frequency: '0 */3 * * *'
+        frequency: 'H */3 * * *'
         job-name: maintenance-pull-janitor
         repo-name: 'k8s.io/test-infra'
-
+    - aws-janitor:
+        branch: master
+        frequency: 'H * * * *'
+        job-name: maintenance-aws-janitor
```

Comment:
emmm... presubmit assumes this to match `maintenance-pull-*`, guess either rename it or move it to https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/bootstrap-maintenance-ci.yaml, to make travis green.